packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -370,6 +371,9 @@ const Dropdown = ({
                         id='dropdown-display'
                         ref={dropdown_ref}
                     >
+                        {className === 'dropdown-statement-filter' && (
```

Comment:
Same here, can you rewrite this as a parameter or prop instead of doing this condition?